### ISSUE TITLE
Accessibility testing—missing description and potential contrast issue

### ISSUE THREAD

**Issue by jenn-sweet** [CajSWehst/web-design-2-independent-project#11]
Issue: Missing title description on about and portfolio 
Issue: Wave identified potential issues on the contact page with contrast
